glitter/page.py

```diff
@@ -39,15 +39,23 @@ def render(self, rerender):
         # Add some classes to the block to help style it
         block_classes = self.css_classes()
 
-        # Render the block
```

Comment:
Think this needs changing. If the block doesn't exist, we still probably want to use the blocks chosen render function.

Something like:

```
block_class = self.content_block.content_type.model_class()
render_function = block_class.render_function
```

Only downside... we'll have to check all existing blocks if they've got a custom render function.